src/components/Drawer/styles.js

```diff
@@ -0,0 +1,70 @@
+import styled from '@emotion/styled'
+import { createMuiTheme } from '@material-ui/core/styles'
+import { theme as localTheme } from '~/utils/styles'
+import Text from '~/components/Utilities/Text'
+
+export const DrawerItem = styled(Text)`
+  /* keeping it for later when I will add the text for the drawer menu as an example */
+`
+
+export const DrawerButtons = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  height: 24px;
+  position: absolute;
+  left: 56%;
+  top: 5%;
```

Comment:
You can adjust this in the header branch but unless you are centering the element dead center like so

`top: 50%, left: 50%` and `transform: translate(-50%, -50%)` avoid using % value for absolute positioned element, this will create inconsistency responsive wise. Fix value like top: 20px and left: 20px is better